cloud-sql/sqlserver/activerecord/README.md

```diff
@@ -0,0 +1,88 @@
+# Connecting to Cloud SQL - MySQL
+
+## Before you begin
+
+1. If you haven't already, set up a Ruby Development Environment by following the [ruby setup guide](https://cloud.google.com/ruby/docs/setup) and 
+[create a project](https://cloud.google.com/resource-manager/docs/creating-managing-projects#creating_a_project).
+
+1. Create a 2nd Gen Cloud SQL Instance by following these 
```

Comment:
nit: First gen is officially deprecated, so we can finally drop. Not super important, but something we should remember to do elsewhere when we refresh the READMEs.
```suggestion
1. Create a Cloud SQL Instance by following these 
```